cmd/juju/crossmodel/list.go

```diff
@@ -96,6 +114,8 @@ func (c *listCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.ModelCommandBase.SetFlags(f)
 	f.StringVar(&c.applicationName, "application", "", "return results matching the application")
 	f.StringVar(&c.interfaceName, "interface", "", "return results matching the interface name")
+	f.StringVar(&c.userName, "user", "", "return results matching the connected user name")
+	f.StringVar(&c.consumerName, "consumer", "", "return results matching the consumer user name")
```

Comment:
these help strings don't feel like they really disambiguate when user should be used vs consumer. And given my intuition was the opposite of what you programmed, we probably should be clearer here.

Reply:
reworded the help